drivers/ethernet/eth_stm32_hal.c

```diff
@@ -32,18 +32,112 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 #error DTCM for DMA buffer is activated but zephyr,dtcm is not present in dts
 #endif
 
+static HAL_StatusTypeDef ETH_RD_PHY_REG( ETH_HandleTypeDef* ps_heth,
```

Comment:
Function name does not follow convention.

Reply:
> Function name does not follow convention.

renamed to `read_eth_phy_register`